gradle/dependencies.gradle

```
@@ -11,7 +11,7 @@ def versions = [
     jetbrainsAnnotations  : '13.0',
     junit                 : '4.12',
     kotlin                : '1.3.71',
-    kotlinCoroutines      : '1.3.1',
+    kotlinCoroutines      : '1.3.5-native-mt',
```

Do we need the `native-mt` version, I think I read somewhere that ktor might not be compatible with it ?

---

Just a copy paste. It might be the reason why iOS sample does not return results. Will try without
```suggestion
    kotlinCoroutines      : '1.3.5',
```